caldav/caldav.go

```diff
@@ -58,3 +58,8 @@ type CalendarObject struct {
 	ETag    string
 	Data    *ical.Calendar
 }
+
+// IsEmpty tests if a CalendarCompRequest contains any information
+func (z CalendarCompRequest) IsEmpty() bool {
```

Comment:
Maybe we can just check `z.Name == ""` and maybe we don't need to introduce `IsEmpty`? My reading of [the RFC](https://tools.ietf.org/html/rfc4791#section-9.6.1) suggests it's actually the only required field.

Reply:
remove IsEmpty